src/main/java/no/nav/veilarboppfolging/controller/OppfolgingController.java

```diff
@@ -168,13 +168,21 @@ public VeilederTilgang hentVeilederTilgang(@RequestParam("fnr") String fnr) {
         return oppfolgingService.hentVeilederTilgang(fnr);
     }
 
+    @GetMapping("/hentPeriode")
+    public OppfolgingPeriodeMinimalDTO hentOppfolgingsPeriode(@RequestParam(value = "fnr", required = false) String fnr, @PathVariable String uuid){
```

Comment:
Hvis man bruker PathVariable så burde det også spesifiseres i URLen. F.eks /periode/{uuid}

Reply:
Ja, det kan jeg gjøre. men når jeg tenker meg om så er det kanskje å bruke RequestBody isteden? Ser at det kanskje er litt rart å bruke en uuid i pathen, eller hva tenker du?

Reply:
GET med body fungerer gjerne ikke så bra, tenker det er innafor å bruke uuid for å referere til en periode

Reply:
En annen ting, hvis man bruker "fnr" fra requestet til å gjøre autentisering så vil det føre til et sikkerhetshull for veiledere siden de kan gjøre requests på et fnr de har tilgang til, men sende med en oppfølgingsperiode på en annen bruker.

Reply:
Det er sant, jeg har retta det nå